config.libsonnet

```diff
@@ -34,6 +34,7 @@
       'pods.json': 'AMK9hS0rSbSz7cKjPHcOtk6CGHFjhSHwhbQ3sedK',
       'statefulset.json': 'dPiBt0FRG5BNYo0XJ4L0Meoc7DWs9eL40c1CRc1g',
     },
+    grafanaDashboardNamePrefix : "K8s / ",
```

Comment:
I really dislike this short name in the dashboard already.
I would prefer to simply write `Kubernetes`.

Reply:
I'm happy to change this default to "Kubernetes /" - it's easy enough to override if consumer wants something more terse